docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -91,25 +82,24 @@ You need the following to get invited to TechPass and complete the onboarding:
   4. Select **Scan a QR code**.
   5. Go back to your computer and click **Next**.
 
-  ![vendor-mfa-2](assets/support/vendor-mfa-2-new.png)
+    ![vendor-mfa-2](assets/support/vendor-mfa-2-new.png)
 
-  6. Scan the QR code on your computer screen and click **Next**. 
+  6. Scan the QR code on your computer screen and click **Next**. Your TechPass account gets activated and linked to the authenticator app.
 
-  Your TechPass account gets activated and is now linked to the authenticator app.
-    ![vendor-scan-qr-code](assets/support/vendor-mfa-3-new.png)
+     ![vendor-scan-qr-code](assets/support/vendor-mfa-3-new.png)
 
-  A number is shown on your browser.
+   A number is shown on your browser.
+   
+    ![number-mfa](assets/images/onboarding/po-non-se/number-mfa.png)
 
-   ![number-mfa](assets/images/onboarding/po-non-se/number-mfa.png)
-  
   7. On the Authenticator app, enter the number shown, and select **Yes** to authenticate your sign-in. 
    
-   ![vendor-confirmed-mfa](assets/support/vendor-mfa-5-new.png)
+    ![vendor-confirmed-mfa](assets/support/vendor-mfa-5-new.png)
 
   8. On your computer, click **Next**.
   9. Choose the country code, enter your mobile phone number and click **Next**.
   
-  ![vendor-mfa-6](assets/support/vendor-mfa-6-new.png)
+    ![vendor-mfa-6](assets/support/vendor-mfa-6-new.png)
 
   You will receive a six-digit code on this phone number. 
```

Comment:
This looks out of place in the documentation.

"this phone number" sounds a little odd. I have copied what Microsoft has for reference.

"5. Type the code sent to you through text message to your mobile device, and then select Next. The page changes to show your success."

https://support.microsoft.com/en-us/account-billing/set-up-text-messaging-as-your-verification-method-a61dcc0c-a003-424f-8a64-d92534ba0dad

Reply:
fixed.